x-pack/plugins/observability_solution/apm/common/entities/types.ts

```diff
@@ -5,6 +5,19 @@
  * 2.0.
  */
 
+interface TraceMetrics {
+  latency?: number | null;
+  throughput?: number | null;
+  failedTransactionRate?: number | null;
+}
+
+interface LogsMetrics {
+  logRate?: number | null;
+  logErrorRate?: number | null;
+}
+
+export type EntityMetrics = TraceMetrics & LogsMetrics;
+
```

Comment:
Can you remove this? I think it was a merge problem I deleted all entity metrics definitions on another PR.

```suggestion

```